meta-refkit/recipes-ros/moveit/moveit-ros-planning/0002-moveit_ros-planning-Use-TinyXML2-instead-of-TinyXML.patch

```diff
@@ -0,0 +1,83 @@
+From 2795d7515acfbd23723429ee5b611de88404d324 Mon Sep 17 00:00:00 2001
+From: Dmitry Rozhkov <dmitry.rozhkov@linux.intel.com>
+Date: Wed, 25 Jan 2017 11:06:58 +0200
+Subject: [PATCH 2/2] moveit_ros/planning: Use TinyXML2 instead of TinyXML
+
+Upstream-Status: Inappropriate [TinyXML2 is WONTFIX in indigo-devel]
```

Comment:
Add a link here to the issue. At least I'm interested if we can ever upstream this... I don't even know what is `indigo-devel`.